packages/patternfly-4/react-core/src/components/Checkbox/Checkbox.js

```diff
@@ -44,6 +47,7 @@ class Checkbox extends React.Component {
           onChange={this.handleChange}
           aria-invalid={!isValid}
           disabled={isDisabled}
+          checked={isChecked}
```

Comment:
This will be breaking so we need to gracefully do this by using checked or isChecked. We can depracate it in next major release.

Reply:
We are still in alpha so I think breaking is fine.  Think it should only be isChecked.